vllm/model_executor/models/olmo.py

```diff
@@ -85,13 +85,12 @@ def __init__(
         self.clip_qkv = config.clip_qkv
 
         # Attention input projection. Projects x -> (q, k, v)
-        self.qkv_proj = QKVParallelLinear(
-            self.hidden_size,
-            self.head_dim,
-            self.total_num_heads,
-            bias=config.attention_bias,
-            quant_config=quant_config,
-        )
+        self.qkv_proj = QKVParallelLinear(self.hidden_size,
+                                          self.head_dim,
+                                          self.total_num_heads,
+                                          bias=config.attention_bias,
+                                          quant_config=quant_config,
+                                          prefix=f"{prefix}.qkv_proj")
```

Comment:
```suggestion
                                          prefix=f"{prefix}.qkv_proj"，)
```
Let's add a comma here to avoid unnecessary whole lines formatting.